subprojects/docs/src/docs/userguide/workingWithDependencies.adoc

```diff
@@ -46,6 +46,18 @@ You can iterate over the complete set of artifacts resolved for a module with th
 Iterating over the artifacts of a module automatically resolves the configuration. A resolved configuration becomes immutable and cannot add or remove dependencies. If needed you can copy a configuration for further modification via api:org.gradle.api.artifacts.Configuration#copy()[].
 ====
 
+=== Navigating the dependency graph
+
+As a plugin developer, you might be faced with the challenge of navigating the full graph of dependencies assigned to a configuration e.g. for turning the dependency graph into a visualization. You can access the full graph of dependencies for a configuration with the help of the api:org.gradle.api.artifacts.result.ResolutionResult[].
```

Comment:
⭕️ "faced with the challenge" sounds odd to me - the API makes this super simple :)

Reply:
done